File: Objective-C Sample Code/Chapter 5/Going Further/Top10/Top10 WatchKit App Extension/MainRow.m
//
//  MainRow.m
//  Top10
//
//  Created by Cory Bohon on 12/11/16.
//  Copyright © 2016 Cory Bohon. All rights reserved.
//

#import "MainRow.h"

@implementation MainRow

- (void)configureWithJSONItem:(JSONItem *)item atIndex:(NSUInteger)index
{
    [self.image setImage:[UIImage imageNamed:@"MusicNote"]];
    [self.label setText:[NSString stringWithFormat:@"%u - %@", index + 1, item.songTitle]];
}

@end

File: Swift Sample Code/Chapter 11/Top10/Top 10 WatchKit App Extension/MainRow.swift
//
//  MainRow.swift
//  Top10
//
//  Created by Cory Bohon on 2/13/17.
//  Copyright © 2017 Cory Bohon. All rights reserved.
//

import UIKit
import WatchKit

class MainRow: NSObject {
    
    @IBOutlet weak var image: WKInterfaceImage?
    @IBOutlet weak var label: WKInterfaceLabel?
    
    func configure(item: JSONItem, index: Int) {
        self.image?.setImage(UIImage(named: "MusicNote"))
        self.label?.setText("\(index + 1) - \(item.songTitle!)")
    }
}
